criu/proc_parse.c

```diff
@@ -303,6 +304,23 @@ static int vma_get_mapfile_user(const char *fname, struct vma_area *vma,
 	}
 
 	vfi_dev = makedev(vfi->dev_maj, vfi->dev_min);
+
+	if (is_memfd(vfi_dev, fname)) {
+		struct fd_link link;
+		link.len = strlen(fname);
+		strcpy(link.name, fname);
```

Comment:
I think that it is safer to use `strlcpy` here. Sorry, I'm have a little paranoia. ;)
`strcpy(link.name, fname, sizeof(link.name));`

Reply:
Yes, I agree, fixed